pointcloud/basic.go

```diff
@@ -51,3 +53,21 @@ func (cloud *basicPointCloud) Set(p r3.Vector, d Data) error {
 func (cloud *basicPointCloud) Iterate(numBatches, myBatch int, fn func(p r3.Vector, d Data) bool) {
 	cloud.points.Iterate(numBatches, myBatch, fn)
 }
+
+// CloudFromVectors converts a list of r3.Vectors into a pointcloud with the specified color.
+func CloudFromVectors(vectors []r3.Vector, c color.NRGBA) (PointCloud, error) {
```

Comment:
[nit] 'PointCloudFromVectors` would be better

Reply:
I agree. I also feel like this function should remain exported.
The issue the linter raised with the name `PointCloudFromVectors` is that the exported function would be called `pointcloud.PointCloudFromVectors` which would cause a "stutter".
Hence, I will keep the name of the function here and update `TestPointCloudFromVectors` to `TestCloudFromVectors`.
chatted offline with Ray abt this and this is the consensus we came to.

Reply:
To avoid the stutter, `VectorsToPointCloud` is also an option but I don't have a strong opinion here